app/payment/session.go

```diff
@@ -69,7 +69,7 @@ func OpenPayCh(pctx context.Context,
 	openingBals perun.BalInfo,
 	challengeDurSecs uint64) (PayChInfo, error) {
 	paymentApp := perun.App{
-		Def:  ppayment.AppDef(),
+		Def:  ppayment.NewApp(),
```

Comment:
Looking at the commit message, I think this change belongs to the [previous commit](https://github.com/hyperledger-labs/perun-node/pull/110/commits/88b3a7ff60ebb42d8f8f0ab10ada34ae97d0da99) that is related to change in `go-perun`.  Can you move the `payment.AppDef()` changes (here and all places below) to that commit.

So that one contains changes related only to the use of Transactor interface ?

Reply:
Moved to this [commit](https://github.com/hyperledger-labs/perun-node/pull/110/commits/f955e9f33a5a09559f837b42cee4bc925859112e)